src/codegen/wasm.zig

```diff
@@ -1443,6 +1447,12 @@ pub const Context = struct {
 
         return WValue{ .local = struct_ptr.multi_value.index + @intCast(u32, extra.data.field_index) };
     }
+    fn airStructFieldPtrIndex(self: *Context, inst: Air.Inst.Index, index: u32) InnerError!WValue {
+        const ty_op = self.air.instructions.items(.data)[inst].ty_op;
+        const struct_ptr = self.resolveInst(ty_op.operand);
+
+        return WValue{ .local = struct_ptr.multi_value.index + index };
```

Comment:
Let's wire these functions together since there is shared logic (even though it is pretty simple)